examples/with-eip-1193-provider/src/lib/turnkey.ts

```diff
@@ -0,0 +1,176 @@
+"use server";
+
+import {
+  ApiKeyStamper,
+  type TApiKeyStamperConfig,
+} from "@turnkey/api-key-stamper";
+import { TurnkeyClient, createActivityPoller } from "@turnkey/http";
+import { Attestation, Email, PassKeyRegistrationResult } from "./types";
+
+import { generateRandomBuffer, base64UrlEncode } from "./utils";
+import {
+  ETHEREUM_WALLET_DEFAULT_PATH,
+  PUBKEY_CRED_TYPE,
+  ALG_ES256,
+} from "./constants";
+import { UUID } from "crypto";
+import { Address } from "viem";
+
+const {
+  TURNKEY_API_PUBLIC_KEY,
+  TURNKEY_API_PRIVATE_KEY,
+  NEXT_PUBLIC_ORGANIZATION_ID,
+  NEXT_PUBLIC_TURNKEY_RPID,
+} = process.env;
+
+export const registerPassKey = async (
+  email: Email
+): Promise<PassKeyRegistrationResult> => {
+  if (!NEXT_PUBLIC_TURNKEY_RPID) {
+    throw "Error must define NEXT_PUBLIC_TURNKEY_RPID in your .env file";
+  }
```

Comment:
I think this should use NEXT_PUBLIC_WEBAUTHN_RPID? Why the switch to `NEXT_PUBLIC_TURNKEY_RPID`?